mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -14,16 +17,20 @@ CLASS net/minecraft/unmapped/C_usxaxydn net/minecraft/entity/LivingEntity
 	FIELD f_ffcustbq bodyTrackingIncrements I
 	FIELD f_figrybhh noDrag Z
 	FIELD f_fqkpwpyn handSwingProgress F
+	FIELD f_gbyxgptg TICKS_BEFORE_USE_EFFECTS I
```

Comment:
Is 'use' here a verb (the effects are being 'use'd) or an adjective (the type of effect is 'use')?
If it's an adjective then this seems like a good name, but if it's a verb I think
```suggestion
	FIELD f_gbyxgptg TICKS_BEFORE_EFFECTS_USED I
```
would be clearer and more grammatically correct.

Reply:
how about this?
```suggestion
	FIELD f_gbyxgptg ITEM_USAGE_EFFECTS_DELAY I
```